creation/web_base/condor_vars.lst

```diff
@@ -106,9 +108,11 @@ X509_SKIP_HOST_CHECK_DNS_REGEX	C	-	+				N	N	-
 GLIDEIN_LOCAL_TMP_DIR   C       -               +                               Y       N       @
 TMP_DIR			S	-		GLIDEIN_Tmp_Dir			Y	Y	@
 PUBLISH_OBITUARIES      C       False           +                               N       N       -
-SiteWMS_WN_Draining  C   False       +   N   Y   -
+SiteWMS_WN_Draining C   False       +   N   Y   -
 SiteWMS_WN_Preempt  C   False       +   N   Y   -
 GLIDEIN_PROVIDES    S	-			+   N	Y	+
+GLIDEIN_PROVIDES    S	-           +   N   Y   +
```

Comment:
This seems a duplicate line not needed

Reply:
did not see this before, thanks for catching!